crates/ruff_python_ast/src/helpers.rs

```diff
@@ -1213,42 +1213,37 @@ pub struct SimpleCallArgs<'a> {
 }
 
 impl<'a> SimpleCallArgs<'a> {
-    pub fn new(args: &'a [Expr], keywords: &'a [Keyword]) -> Self {
-        let mut result = SimpleCallArgs::default();
+    pub fn new(
+        args: impl IntoIterator<Item = &'a Expr>,
+        keywords: impl IntoIterator<Item = &'a Keyword>,
+    ) -> Self {
+        let args = Vec::from_iter(
```

Comment:
Using `from_iter` is potentially faster than adding element by element because `from_iter` can use `Iterator::size_hint` to *guess* the necessary capacity.